sdk/keyvault/Azure.Security.KeyVault.Secrets.Extensions.Configuration/src/Azure.Security.KeyVault.Secrets.Extensions.Configuration.csproj

```diff
@@ -6,6 +6,7 @@
     <PackageTags>$(PackageTags);azure;keyvault</PackageTags>
     <Version>1.0.0-preview.1</Version>
     <EnableApiCompat>false</EnableApiCompat>
+    <IsExtensionClientLibrary>true</IsExtensionClientLibrary>
```

Comment:
Why on the test projects?

Reply:
Good point, I'll move some packages under IsTestProject condition instead.